src/Decorations.ts

```diff
@@ -18,16 +19,16 @@ export default class Decorations {
   decorations: NamedDecoration[];
   decorationMap: DecorationMap;
 
-  constructor() {
-    const fontWidthPx = 8.04;
+  constructor(fontSize: FontSize) {
+    console.log(fontSize)
     const iconWidthPx = 4;
```

Comment:
do we want this to be fixed or vary with the font size?

Reply:
Hmm unsure.